src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -628,9 +610,10 @@ bool emberAfAccessControlClusterReviewFabricRestrictionsCallback(
     CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
     const Clusters::AccessControl::Commands::ReviewFabricRestrictions::DecodableType & commandData)
 {
-    if (commandPath.mEndpointId != 0)
+    if (commandPath.mEndpointId != kRootEndpointId)
     {
         ChipLogError(DataManagement, "AccessControlCluster: invalid endpoint in ReviewFabricRestrictions request");
+        commandObj->AddStatus(commandPath, Protocols::InteractionModel::Status::InvalidValue);
```

Comment:
InvalidCommand is likely the right error.

Reply:
done